src/WebApps/SnWebApplication.Api.InMem.TokenAuth/Startup.cs

```diff
@@ -56,9 +72,9 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
+                IdentityModelEventSource.ShowPII = true;
             }
-
-            app.UseHttpsRedirection();
+            //app.UseHttpsRedirection();            
```

Comment:
Should we remove this? This is required in other cases, only causes problems in a docker environment, right? Maybe add https redirection based on a config?

Reply:
yes, services may communicate via http between containers directly instead public urls